From: John Jolly <jjolly@suse.de>
Date: Fri, 08 Jul 2014 15:52:20 -0600
Subject: kabi: Expose dependent function necessary for previous path
Patch-mainline: Never, kabi
References: bnc#886321

In order to successfully compile the zfcpdump kernel, the net_secret 
function needed to be exposed consistent to the exposure of the 
secure_dccp_sequence_number function.

This patch fixes the following build error:
../net/core/secure_seq.c: In function 'secure_dccp_sequence_number':
../net/core/secure_seq.c:164:2: error: implicit declaration of function 'net_secret_init' [-Werror=implicit-function-declaration]
  net_secret_init();
  ^
../net/core/secure_seq.c:168:12: error: 'net_secret' undeclared (first use in this function)
  hash[3] = net_secret[15];

Signed-off-by: John Jolly <jjolly@suse.de>

--- linux-3.12-SLE12.orig/net/core/secure_seq.c	2014-07-05 11:02:42.000000000 -0600
+++ linux-3.12-SLE12/net/core/secure_seq.c	2014-07-05 11:02:53.000000000 -0600
@@ -10,7 +10,7 @@
 
 #include <net/secure_seq.h>
 
-#if IS_ENABLED(CONFIG_IPV6) || IS_ENABLED(CONFIG_INET)
+#if IS_ENABLED(CONFIG_IPV6) || IS_ENABLED(CONFIG_INET) || 1
 #define NET_SECRET_SIZE (MD5_MESSAGE_BYTES / 4)
 
 static u32 net_secret[NET_SECRET_SIZE] ____cacheline_aligned;
